source/psp/video_hardware_images.cpp

```diff
@@ -990,7 +990,7 @@ loadimagepixels
 =============
 */
 										// HACK HACK HACK
-byte* loadimagepixels (char* filename, qboolean complain, int matchwidth, int matchheight)
+byte* loadimagepixels (char* filename, qboolean complain, int matchwidth, int matchheight, int type)
```

Comment:
if `type` is used essentially as boolean here, would it be better to make it `bool extendedformats` or something and then later instead of `if (type == 1) return NULL;` it could be `if (!extendedformats) return NULL;`. thought it'd communicate intent better

Reply:
Idea is to (if needed/deemed worthwhile) extent that functionality to, say, adding another integer type to only load TGA graphics for UX